highlevel/robot/entity/type.py

```diff
@@ -33,3 +35,14 @@ class Direction(Enum):
     BACKWARD = 'BACKWARD'
     LEFT = 'LEFT'
     RIGHT = 'RIGHT'
+
+
+class MotionResult(Enum):
```

Comment:
This seems very very specific to the motion control and no way generic to all the robot.

Move this to the controller file?